src/Core/Context/CurrentContext.cs

```diff
@@ -261,7 +261,7 @@ public async Task<bool> OrganizationOwner(Guid orgId)
 
             if (Providers.Any())
             {
-                return (await GetProviderOrganizations()).Any(po => po.OrganizationId == orgId);
+                return (await ProviderUserForOrgAsync(orgId));
```

Comment:
```suggestion
                return await ProviderUserForOrgAsync(orgId);
```

Reply:
I want to blame this on linting, but I'm not sure if I can :D